libs/blocks/marquee/marquee.js

```diff
@@ -132,18 +130,12 @@ export default function init(el) {
   const text = headline.closest('div');
   text.classList.add('text');
   const media = foreground.querySelector(':scope > div:not([class])');
-
   if (media) {
     media.classList.add('media');
+    decorateImage(media);
```

Comment:
Looks like you are calling `decorateImage()` even if it is video. I see it won't append any element if there is no `picture` element so, it should work find for now. Though, I feel this can go `else` of `if (video)` below so we don't call it at all when it is not necessary.